src/smtpmailer/MultipartEncoder.hx

```diff
@@ -32,15 +32,21 @@ class MultipartEncoder {
 		} else {
 			main = content;
 		}
-		
-		main.setHeader('From', email.from);
-		main.setHeader('To', email.to.join(','));
+
+		main.setHeader('From', email.from.displayName != null
```

Comment:
Could the null check and formatting be moved to the abstract? Resulting in something like:
```haxe
main.setHeader('From', email.from.format());
main.setHeader('From', email.to.map(function(to) return to.format()));
```